google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -61,6 +64,15 @@ private Builder(BigQueryOptions options) {
     }
 
     @Override
+    public Builder setTransportOptions(TransportOptions transportOptions) {
```

Comment:
1. That defeats the entire purpose of this change
2. In the future, Datastore will support both http & grpc, so its implementation of this function would not actually reject either of those transport option types.